src/services

```diff
@@ -1069,3 +1069,46 @@ p({
     }
   """
 })
+
+p({
```

Comment:
same, move these into src/foam/i18n/services.jrl
Reduces code merge conflicts in the future.